examples/IRMQTTServer/IRMQTTServer.ino

```diff
@@ -381,6 +380,9 @@ using irutils::msToString;
 #endif  // REPORT_VCC
 
 // Globals
+
+
+
```

Comment:
Nitpick: These blank lines can be removed.

Reply:
Already edited. :+1: